.gitmodules

```diff
@@ -71,4 +71,6 @@
 	url = https://github.com/pytorch/cpuinfo.git
 [submodule "cmake/external/onnx-tensorrt"]
 	path = cmake/external/onnx-tensorrt
-	url = https://github.com/onnx/onnx-tensorrt.git
+	url = https://github.com/symphonylyh/onnx-tensorrt.git
+	branch = deberta_trt_plugin
```

Comment:
Current ORT master is pointing to this parser commit (https://github.com/onnx/onnx-tensorrt/tree/4f54a1950e1174dca490900eb7b07cc374f53d41), which is based on the parser 8.2-GA release. I think we should cherry pick your DeBERTa parser change on top of it. ORT master will not point to parser master because it's for the future TRT8.4 release.

Reply:
That's a good point. Yesterday they only cherry pick to the parser master. I'll ask them to cherry pick to 8.2-GA release branch as well, and then ORT master can do a submodule update I guess?

Reply:
Yes. Please check with Kevin how this was dealt for the commit (https://github.com/onnx/onnx-tensorrt/tree/4f54a1950e1174dca490900eb7b07cc374f53d41) before. We should follow the same way for the DeBERTa parser change.

Reply:
I've cherry-picked the changes to 8.2-GA, you should be able to point to the 8.2-GA branch now